test/system/102-create.bats

```diff
@@ -7,14 +7,14 @@ load helpers
 }
 
 @test "Create a container with a valid custom name ('not-running')" {
-  run_toolbox -y create -c "not-running"
+  run_toolbox -y create "not-running"
```

Comment:
We still support the `--container` flag, so we should also continue to test it. :)

Reply:
Ah, right.

Reply:
I updated the test cases to use both ways of creating a named container.